bindata/network/ovn-kubernetes/self-hosted/ovnkube-node.yaml

```diff
@@ -16,6 +16,9 @@ metadata:
     {{ if .OVNIPsecEnable }}
     networkoperator.openshift.io/ipsec-enabled: "true"
     {{ end }}
+    {{ if .DefaultMasqueradeNetworkCIDRs }}
+    networkoperator.openshift.io/default-masquerade-network-cidrs: "{{.DefaultMasqueradeNetworkCIDRs}}"
```

Comment:
why is it necessary to annotate the DS? I see you use it in bootstrap to see if there was already a value set. Why is that necessary if we have the default/API values already?

Also, why is this only done in self-hosted and not managed?

Reply:
> why is it necessary to annotate the DS? I see you use it in bootstrap to see if there was already a value set. Why is that necessary if we have the default/API values already?

On new clusters we need to set the annotation so the default values are not wiped on the next reconciliation.
Consider this example:
1. Fresh cluster starts up, CNO configures the default masq subnets
2. OVN-k daemonset is rolled out with the new default masq subnets
3. On next reconciliation we need to ensure we preserve the defaults from step 1